app/webrtc/client/tabBar.tsx

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+
+import { useSetting } from '../../../client/contexts/SettingsContext';
+import { addAction } from '../../../client/views/room/lib/Toolbox';
+import { useTranslation } from '../../../client/contexts/TranslationContext';
+
+addAction('webRTCVideo', () => {
+	const t = useTranslation();
+	const enabled = useSetting('WebRTC_Enabled') && (useSetting('Livechat_call_provider') === t('WebRTC'));
+
+	const handleClick = () => { window.open('/', '_blank'); };
+
+	return useMemo(() => (enabled ? {
+		groups: ['live'],
+		id: 'webRTCVideo',
+		title: 'WebRTC_Call',
+		icon: 'phone',
+		action: handleClick,
+		full: true,
+		order: 4,
+	} : null), [enabled, t]);
```

Comment:
![image](https://user-images.githubusercontent.com/34130764/124385966-99c73680-dcf5-11eb-8793-99bdf8c7af60.png)
New warning getting introduced bcs of this code. Please resolve them